tests/post/steps/test_volume.py

```diff
@@ -255,6 +285,24 @@ def _check_status():
         name='checking error for Volume {}'.format(name)
     )
 
+
+@then(parsers.parse("the Volume '{name}' is marked for deletion"))
+def check_volume_deletion_timestamp(name, k8s_custom_client):
+    def _check_deletion_timestamp():
+        volume = _get_volume(k8s_custom_client, name)
+        if volume is None:
+            assert False, 'Volume {} not found'.format(name)
+
+        assert volume['metadata']['deletionTimestamp'] is not None,\
+            'Unexpected deletion timestamp got {}'.format(
```

Comment:
How is it `Unexpected deletion timestamp` when the point of the test is that you expect one?

Can be rewritten as:

```python
    def _check_volume_deletion_timestamp():
        volume = _get_volume(k8s_custom_client, name)
        assert volume is not None, 'Volume {} not found'.format(name)
        assert volume['metadata']['deletionTimestamp'] is not None,\
            'Volume {} is not marked for deletion'.format(name)
```